Docs: Refactor all business features into feature reference + change "privacy level" page (Diátaxis) (#10007)
readthedocs/readthedocs.org: docs/user/commercial/index.rst

About |com_brand|

> =================

Reviewer: Does this work? I never know if it's going to expand first or not.

Author: This one worked, but we can't write `*|com_brand|*` :facepalm: 

![image](https://user-images.githubusercontent.com/374612/218106177-2bc8bada-5cd9-432b-a0d5-41c088f98f3a.png)

Reviewer: Yea... could be a good use case for myst, but not sure if they have substitutions heh :)

.. this page is currently moving towards becoming "About Read the Docs for Business"
.. rather than an index of features.

We offer *|com_brand|*
for building and hosting commercial documentation.

Our commercial solutions are provided as a set of subscriptions that are paid and managed through an online interface.
In order to get started quickly and easily, a `trial option <https://about.readthedocs.com/pricing/>`__ is provided free of charge.

.. seealso::

>    `Read the Docs website: Features <https://about.readthedocs.com/about/>`__
>       A general presentation of our features is available on our website, and the
>       `pricing page <https://about.readthedocs.com/pricing/>`__ lists all details and features for each subscription level.
> 
>    `Read the Docs website: Company presentation <https://about.readthedocs.com/company/>`__
>       Information about the company running Read the Docs, including our mission, team, and community.

Reviewer: ```suggestion
   `Read the Docs website: Features <https://about.readthedocs.com/about/>`__
      A high-level overview of platform features is available on our website, and the
      `pricing page <https://about.readthedocs.com/pricing/>`__ has a feature breakdown by subscription level.

   `Read the Docs website: Company <https://about.readthedocs.com/company/>`__
      Information about the company running Read the Docs, including our mission, team, and community.
```

I don't love the `Read the Docs website` prefix, but like you, couldn't come up with anything better. I want something other than `RTD` on it tho, like `Company website: Features` or something? But this is fine for now.

Author: "Company website: Company" isn't great but "Company website: About" works well.

I'm not sure that it's nice in all cases, since presenting "Read the Docs" as a company or a platform or a community are all aspects of the website.

If we could find a nice "universal" way to refer to our website, we could put it in the style guide. I unfortunately also don't have any good ideas right now :(

Reviewer: > I'm not sure that it's nice in all cases, since presenting "Read the Docs" as a company or a platform or a community are all aspects of the website.

Yea, definitely part of the complexity. The same word being multiple things is always going to lead to confusion.

Commercial documentation solutions
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

In addition to providing the same features as |org_brand|,

> commercial subscriptions to Read the Docs add additional features based on the same software platform but on a separate infrastructure.

Reviewer: ```suggestion
commercial subscriptions to Read the Docs add additional features and run on separate infrastructure.
```

.. figure:: /img/community_in_business.png
   :align: center
   :scale: 50%

   Read the Docs for Community and Business are a combined system:

>    All features developed for community benefit the business solution, and a lot of solutions developed for business

Reviewer: ```suggestion
   All features developed for community benefit the business solution, and most solutions developed for business users
```

>    are implemented for community.

Reviewer: ```suggestion
   are implemented for the community.
```

I feel like using `business` and `community` as shorthand doesn't really work.

Author: no, you're right!

The following list is a high-level overview of the areas covered by |com_brand|.

> If you want exact details, please refer to the `pricing page <https://about.readthedocs.com/pricing/>`__ of the public site.

Reviewer: ```suggestion
If you a full feature breakdown, please refer to our `pricing page <https://about.readthedocs.com/pricing/>`__.
```

Additional build resources
    Do you have a complicated build process that uses large amounts
    of CPU, memory, disk, or networking resources?

>     Commercial subscriptions offer more resources
>     which results in faster documentation build times.
>     Resources can be increased further for very demanding projects.

Reviewer: ```suggestion
    Commercial subscriptions offer more resources
    which results in faster documentation build times.
    We can also customize builders, such as with a GPU or multiple CPUs.
```

Author: Super! I really didn't want to remove it, but I also didn't like the fact that the previous sentence kind of implied that we are running things artificially slowly.. like an Intel Celeron CPU sold with disabled circuits :)

Author: More clarity FTW!

Reviewer: It's a win/win, an upsell and more explicit :)

Priority support
    We have a dedicated support team that responds to support requests during business hours.

Advertising-free
    All commercially hosted documentation is always ad-free.

Business features

>     A number of extra features exist specifically for business needs, such as
>     organization support with team management, single-sign on, and audit logging.

Reviewer: ```suggestion
    Enjoy additional functionality specifically for companies, such as
    organization support with team management, single-sign on, and audit logging.
```

Author: Definitely nit-picking this one :)

As a native English speaker, how do you read the word "company"?

I read it as a legal entity that is exclusionary to education, NGOs etc. But they can actually benefit from business features.

Reviewer: Hrm, good point. But `business needs` is an odd phrasing that I was trying to remove.

I'd usually use something like `larger organizations`, but that was awkward with the follow sentence, but that can perhaps be fixed?

```
    Enjoy additional functionality specifically for larger organizations, such as
    team management, single-sign on, and audit logging.
```

    .. seealso::

       - :doc:`/commercial/organizations`
       - :doc:`/commercial/single-sign-on`
       - :doc:`/security-log`

> .. _readthedocs.org: https://readthedocs.org
> .. _readthedocs.com: https://readthedocs.com

Reviewer: ```suggestion
```

🔥

Author: Nice find! I kind of wish that Sphinx would have told me this...

Reviewer: Yea, would really love a better rst/sphinx linter.

Author: I was wondering if it's possible to write an upstream issue quickly... but with 1.1k issues open (and 101 PRs) and a lack of response to critical bugs, I'm just not gonna do it :(
